content/hi/docs/reference/glossary/applications.md

```diff
@@ -0,0 +1,12 @@
+---
+title: एप्लिकेशंस (Applications)
+id: applications
+date: 2019-05-12
+full_link:
+short_description: >
+ लेयर वह परत है जहाँ विभिन्न कंटेनराइज्ड एप्लिकेशंस रन करते है।
+aka:
+tags:
+- fundamental
+---
+ लेयर वह परत है जहाँ विभिन्न कंटेनराइज्ड एप्लिकेशंस रन करते है।
```

Comment:
```suggestion
 वह परत जहाँ विभिन्न कंटेनराइज्ड एप्लिकेशंस रन करते हैं।
```

Reply:
@niranjandarshann  Sure! updated as per your suggestion, Thanks!!

Reply:
@niranjandarshann Can you review this as well, I have made necessary changes as suggested by you!